README.md

```diff
@@ -1,18 +1,115 @@
 
-This option is recommended for most users. You will get a working system with all the binaries in place that will do the job. Updating the dependencies isn't automated though; you'll need to go back to each project's respective download page again and manually check if there's an update. As the dependencies aren't updated that often, this shouldn't really be a problem.
+1. Create `C:\bin` and [add it to your path](#adding-a-folder-to-your-path)
+1. Download a Windows build of [mpv](https://mpv.io/) from [here](https://mpv.srsfckn.biz/)
+1. Extract it to `C:\bin`
+1. Create an alias for mpv:
+    `echo "alias mpv=mpv.exe" >> .bashrc`
 
-### Installing it in the linux subsystem for windows
+If you have your movie files in the Windows file system (so NOT in `~userfolder/AppData/Local/lxss/home`), for example on your desktop, you have to provide the path to it via `/mnt/[drive letter]/path/to/your/file/location`. You can also navigate to the folder in Explorer, then type bash in the address bar to open a bash instance to that location
 
-This method is bleeding edge as in: it is quite cumbersome and very experimental right now. The whole subsystem is in beta status (and will be for the foreseeable future). The spring Windows 10 update is supposed to fix some issues with it so that the need to self-compile everything will be gone, further streamlining and simplifying the process.
-This guide is for the adventurous. It will result in a system that you can easily update (as detailed in the guide) but is hard to set up right now. It is also unclear whether it breaks with an update easily. Not really recommended until the spring update has landed for wide availability.
+### Pre Windows 10 Creators Update and earlier
+In old versions of Windows 10, or in Windows 8.1 and earlier, you either don't have access to Bash on Ubuntu on Windows or the version that's there can't be used to do this. All the dependencies are available for Windows though, you just need to track them down:
+
+- [HandBrakeCLI](https://handbrake.fr/downloads2.php)  
+    Get the latest stable version
+- [ffmpeg](https://ffmpeg.org/download.html), the download page for Windows is [here](http://ffmpeg.zeranoe.com/builds/)  
+   Get the latest stable 'static' release.
+- [mkvtoolnix](https://mkvtoolnix.download/downloads.html#windows), the download page for Windows is [here](https://www.fosshub.com/MKVToolNix.html)  
+    Download the latest portable version. It's packaged as a .7z file, I'd recommend using [7zip](http://www.7-zip.org/download.html) to open these files
+- [mpv](https://mpv.io/), the download page for Windows is [here](https://mpv.srsfckn.biz/)  
+    Get the latest version
+- [MP4v2](https://code.google.com/archive/p/mp4v2/)  
+    Functional Windows binaries for this are hard to find, I've been using [this one](http://forum.doom9.org/showthread.php?t=171038). If anyone knows of a more official binary, please let me know
```

Comment:
I think we should make the "please let me know" either a link to a new bug report or ot your email/twitter, so people could actually figure out quickly who,pm to contact (you).